### PR TITLE
added print out for step up event

### PR DESCRIPTION
**Pull request by jasmith-lp**

### CODE REVIEW

File: examples/agent-bot/MyCoolAgent.js [hunk 1]
```diff
@@ -73,6 +74,19 @@ class MyCoolAgent extends Agent {
                         });
                     });
                     this.subscribeMessagingEvents({dialogId: change.result.convId});
+                } else if(change.type === 'UPSERT' && openConvs[change.result.convId] && change.result.conversationDetails.participants.filter(p => p.role === 'CONSUMER')[0].id !== this.consumerId) {
+                    // Step Up...
```
Comment (tasoili) [inline examples/agent-bot/MyCoolAgent.js]: Will API consumers understand what 'step up' means?

Is it feasible that one day we may support a consumer (for example) logging in as one user, then logging out before logging in as another?

```suggestion
                    // ConsumerID changed. Typically, a Step Up from an unauthenticated to an authenticated user.
```

Comment (jasmith-lp) [reply examples/agent-bot/MyCoolAgent.js]: Sounds like you are describing step down.
However, this is unlikely to be supported at a conversation level IMO.

The suggested comment change makes sense to me.

**Comment by jasmith-lp** [comment]
@tasoili can this be merged?
It will be handy for customers looking to implement Step Up with bots.

There are several in APAC looking to integrate their Virtual Assistants with this flow.

**Comment by tasoili** [comment]
Sure, sorry I was under the impression it already got merged.

**Pull request merged**